test/CadlRanchProjects/encode/bytes/src/Generated/RequestBody.cs

```diff
@@ -161,7 +161,7 @@ public virtual async Task<Response> OctetStreamAsync(BinaryData value, Cancellat
             Argument.AssertNotNull(value, nameof(value));
 
             RequestContext context = FromCancellationToken(cancellationToken);
-            using RequestContent content = RequestContentHelper.FromObject(value);
+            using RequestContent content = value;
```

Comment:
We don't need this line. If can implicitly cast to RequestConent, just pass the parameter in the protocol method call.

Reply:
We can't. We need `using`.